components/great-expectations/validate/CSV/component.py

```diff
@@ -3,7 +3,7 @@
 
 def validate_csv_using_greatexpectations(
     csv_path: InputPath(),
-    expectation_suite_path: InputPath(),
+    expectation_suite_path: str,
```

Comment:
why is this change needed?

Reply:
It is needed to correctly pass pipeline arguments to this component. They are passed as file in case of `InputPath`.

Reply:
What kind of "path" is this? Is it a URI? Let's be clear.
`expectation_suite_uri`

Reply:
Your original version has some advantage over this new version.
With the old version I can process/transform a dataset inside the pipeline and then run the validation. With the proposed change, I can no longer do that - I need to first upload the dataset to some cloud storage (Which one? How do I authenticate?)

Reply:
Maybe it is better to download the Great Exception `suite` in a separate step in which `open` or any other package to fetch it could be use, similar to how it is set up today? That would result in two steps in a pipeline but give the user a lot more freedom.

Reply:
@NikeNano @Ark-kun sure, you're right, thanks for the review. I'l rollback this change and this PR will fix only a couple of typos that we're left from the previous PR.